src/Definition/Common.php

```diff
@@ -24,11 +32,17 @@ class Common
      * @param string $length
      * @param bool $nullable
      */
-    public function __construct($type, $length = null, $nullable = true)
+
+    public function __construct($type, $length = null, $nullable = true, $default = null, $format = null)
```

Comment:
? as it is, the `Redshift` and `Snowflake`  classes aren't supporting `default` and `format` so they should be unaffected.